api/access/server.go

```diff
@@ -168,16 +169,71 @@ func (s *Server) GetTransactionResult(ctx context.Context, in *access.GetTransac
 	return nil, errors.New("not implemented")
 }
 
-func (s *Server) GetAccount(ctx context.Context, in *access.GetAccountRequest) (*access.GetAccountResponse, error) {
-	return nil, errors.New("not implemented")
+func (s *Server) GetAccount(_ context.Context, in *access.GetAccountRequest) (*access.GetAccountResponse, error) {
+	height, err := s.index.Last()
+	if err != nil {
+		return nil, fmt.Errorf("could not get height: %w", err)
+	}
+
+	header, err := s.index.Header(height)
+	if err != nil {
+		return nil, fmt.Errorf("could not get header: %w", err)
+	}
+
+	account, err := s.invoker.GetAccount(flow.BytesToAddress(in.Address), header)
+	if err != nil {
+		return nil, err
+	}
+
+	a, err := convert.AccountToMessage(account)
+	if err != nil {
+		return nil, fmt.Errorf("could not convert account to RPC message: %w", err)
+	}
+
+	// For now, we can't just reuse `GetAccountAtLatestBlock` for this because the return types are not the same.
+	resp := access.GetAccountResponse{
+		Account: a,
+	}
+
+	return &resp, nil
 }
```

Comment:
This function is deprecated and the same as the `GetAccountAtLatestBlock`, so I don't understand why you implemented a full function body here, and wrap `GetAccountAtBlockHeight` for the second one?

- `GetAccount` should simply proxy to `GetAccountAtLatestBlock`.
- `GetAccountAtLatestBlock` should use `GetAccountAtBlockHeight` (as it already does)

Reply:
As I wrote: `// For now, we can't just reuse `GetAccountAtLatestBlock` for this because the return types are not the same.` 

GetAccount has the exact same return type, except it's not named the same so we can't use it directly. We'd have to call `GetAccountAtLatestBlock` and then convert its response into the deprecated one. Would it be better this way do you think?